app/views/sessions/_login.html.erb

```diff
@@ -46,7 +46,7 @@
   <% unless has_password %>
     <div class="password-login">
       <%= form_tag '/auth/identity/callback' do %>
-        <label for='auth_key'>Username / Email</label><input type='text' id='auth_key' name='auth_key'/>
+        <label for='auth_key'>Username / Email</label><input type='text' id='auth_key' name='auth_key' autocapitalize="off" autocorrect="off"/>
```

Comment:
https://developers.google.com/web/updates/2015/04/autocapitalize?hl=en
http://www.bennadel.com/blog/2307-disabling-auto-correct-and-auto-capitalize-features-on-iphone-inputs.htm